pkg/console/starter/starter.go

```diff
@@ -175,6 +177,8 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder,
 	)
 
+	configUpgradeableController := unsupportedconfigoverridescontroller.NewUnsupportedConfigOverridesController(operatorClient, ctx.EventRecorder)
```

Comment:
This controller handles reporting our status now, specifically `Upgradeable`, example:

```
status:
  conditions:
  - lastTransitionTime: 2019-03-26T19:31:44Z
    reason: AsExpected
    status: "False"
    type: Failing
  - lastTransitionTime: 2019-03-26T19:37:15Z
    message: 'Progressing: Moving to version 4.0.0'
    reason: Progressing
    status: "True"
    type: Progressing
  - lastTransitionTime: 2019-03-26T19:31:54Z
    reason: AsExpected
    status: "True"
    type: Available
  - lastTransitionTime: 2019-03-26T19:45:24Z
    message: 'Upgradeable: With UnsupportedConfigOverrides: {"auth":{"clientID":"samwisegamgee","logoutRedirect":"https://samwisegamgee.com/you-re-nowt-but-a-ninnyhammer-Sam-Gamgee"},"customization":{"branding":"samwisegamgee"}}'
    reason: AsExpected
    status: "True"
    type: Upgradeable
```